### DISCUSSION TITLE
Atom install on Scientific Linux 7.9

### DISCUSSION THREAD

**Discussion by DarkSky7** [atom/atom#23116] [General]
a "dnf install atom" errors out, so I tried the download and got:
dnf install atom.x86_64.rpm
Scientific Linux $slreleasever - x86_64 - bugfi 0.0  B/s |   0  B     00:00    
Scientific Linux $slreleasever - x86_64 - secur 0.0  B/s |   0  B     00:00    
Scientific Linux $slreleasever - x86_64         0.0  B/s |   0  B     00:00    
Failed to synchronize cache for repo 'sl-fastbugs', ignoring this repo.
Failed to synchronize cache for repo 'sl-security', ignoring this repo.
Failed to synchronize cache for repo 'sl', ignoring this repo.
Last metadata expiration check: 0:31:40 ago on Sat 16 Oct 2021 09:36:17 PM PDT.
Error: 
 Problem: conflicting requests
  - nothing provides libgcrypt.so.20()(64bit) needed by atom-1.58.0-0.1.x86_64
------------------
Is there anything I can do?

**Comment by nadofc781** [comment]
#
